### PR TITLE
Update rotation2d docs and changelog for allwpilib#4611

### PR DESCRIPTION
**Pull request by ohowe1**
For wpilibsuite/allwpilib#4611

I'm not sure if I did the latex math stuff right, though.

### CODE REVIEW

File: source/docs/software/advanced-controls/geometry/pose.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ You can get the distance to another ``Translation2d`` object by using the ``getD
 Rotation
 --------
 
-Rotation in 2 dimensions is representated by WPILib's ``Rotation2d`` class (`Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/math/geometry/Rotation2d.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classfrc_1_1_rotation2d.html>`__). This class has an angle component, which represents the robot's rotation relative to an axis on a 2-dimensional coordinate system. Positive rotations are counterclockwise.
+Rotation in 2 dimensions is representated by WPILib's ``Rotation2d`` class (`Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/math/geometry/Rotation2d.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classfrc_1_1_rotation2d.html>`__). This class has an angle component, which represents the robot's rotation relative to an axis on a 2-dimensional coordinate system. Due to the angle being stored as its sine and cosine components, the angle will always be in the :math:`\left(-\pi, \pi\right]` range. Positive rotations are counterclockwise.
```
Comment (sciencewhiz) [inline source/docs/software/advanced-controls/geometry/pose.rst:16]: Opens with a parentheses and closes with a brace

**Review by sciencewhiz — CHANGES_REQUESTED** [review]

File: source/docs/software/advanced-controls/geometry/pose.rst (hunk 1 above)
Comment (ohowe1) [reply source/docs/software/advanced-controls/geometry/pose.rst:16]: Sorry, I'm not sure I understand?

Comment (sciencewhiz) [reply source/docs/software/advanced-controls/geometry/pose.rst:16]: E32E72E2-59EB-47A3-996B-485E7EDD18E5

Comment (sciencewhiz) [reply source/docs/software/advanced-controls/geometry/pose.rst:16]: Never mind, it's intentional

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**